hw/ip/csrng/dv/cov/csrng_v2s_exclusions.vRefine

```diff
@@ -5,663 +5,18 @@
-y or -libcell option but continues to;	// disable the scoring of Verilog modules defined with the 'celldefine compiler directive.;	set_libcell_scoring;	;	// Enables scoring of SystemVerilog continuous assignments, which is by disabled by default.;	set_assign_scoring;	;	// Scores branches together with block coverage.;	set_branch_scoring;	;	// Scores statements within a block.;	set_statement_scoring;	;	// Enables Toggle scoring and reporting of SystemVerilog enumerations and multidimensional static;	// arrays , vectors, packed union, modport and generate blocks.;	set_toggle_scoring -sv_enum enable_mda -sv_struct_with_enum -sv_modport -sv_mda 16 -sv_mda_of_struct -sv_generate -sv_packed_union;	;	// Enable toggle coverage only on ports.;	set_toggle_portsonly;	;	// Enable scoring of FSM arcs (state transitions).;	// TODO: re-enable this setting, temp disable due to #12544;	// set_fsm_arc_scoring;	;	// Include X-&gt;1|0 for toggle coverage collection. #10332;	set_toggle_includex;	;	// For ternary operator in default SOP mode;	set_expr_scoring -vlog_short_circuit;;// enable coverage on dut and below;;select_coverage -befts -module ${DUT_TOP}...;;// Black-box pre-verified IPs from coverage collection.;deselect_coverage -betfs -module pins_if;deselect_coverage -betfs -module clk_rst_if;deselect_coverage -betfs -module prim_alert_sender...;deselect_coverage -betfs -module prim_alert_receiver...;deselect_coverage -betfs -module prim_count...;deselect_coverage -betfs -module prim_esc_sender...;deselect_coverage -betfs -module prim_esc_receiver...;deselect_coverage -betfs -module prim_onehot_check...;deselect_coverage -betfs -module prim_prince...;deselect_coverage -betfs -module prim_lfsr...;deselect_coverage -betfs -module prim_secded_inv_64_57_dec...;deselect_coverage -betfs -module prim_secded_inv_39_32_dec...;;// Black-box DV CDC module.;deselect_coverage -betfs -module prim_cdc_rand_delay;;// csr_assert_fpv is an auto-generated csr read assertion module. So only assertion coverage is;// meaningful to collect.;deselect_coverage -betf -module *csr_assert_fpv...;select_coverage -assert -module *csr_assert_fpv;;// Only enable assertion coverage;deselect_coverage -betf -module *tlul_assert...;select_coverage -assert -module *tlul_assert;;// Only collect toggle coverage on the DUT and the black-boxed IP (above) ports.;deselect_coverage -toggle -module ${DUT_TOP}...;select_coverage -toggle -module ${DUT_TOP};;select_coverage -toggle -module prim_alert_sender;select_coverage -toggle -module prim_alert_receiver;select_coverage -toggle -module prim_count;select_coverage -toggle -module prim_esc_sender;select_coverage -toggle -module prim_esc_receiver;select_coverage -toggle -module prim_onehot_check;select_coverage -toggle -module prim_prince;select_coverage -toggle -module prim_lfsr;select_coverage -toggle -module prim_secded_inv_64_57_dec;select_coverage -toggle -module prim_secded_inv_39_32_dec;" path="/home/dev/src/hw/dv/tools/xcelium/cover.ccf"></ccf-file>
```

Comment:
Do you know where this line is coming from? Maybe we better remove this if it's not strictly needed.

Reply:
I thought I'd ignore it since it's just in the info branch. But you're right it's probably best to just remove it.